src/predict/api/controllers/prediction.py

```diff
@@ -64,30 +46,13 @@ def get_next_24hr_forecasts(device_channel_id, forecast_start_time):
 
 
 @ml_app.route(api.route['next_1_week_forecasts'], methods=['GET'])
-def get_next_1_week_forecasts(device_channel_id, forecast_start_date):
+def get_next_1_week_forecasts():
     """
     Get forecasts for the next 1 week from specified start day.
     """
     if request.method == 'GET':
-        if type(device_channel_id) is not int:
-            device_channel_id = int(device_channel_id)
-
-        if type(forecast_start_date) is not int:
-            try:
-                forecast_start_date = int(forecast_start_date)
-            except ValueError:
-                error = {
-                    "message": "Invalid prediction start date. expected unix timestamp format like 1500000000",
-                    "success": False}
-                return jsonify(error, 400)
-
-        # change prediction_start_date to datetime format
-        forecast_start_timestamp = dt.datetime.fromtimestamp(
-            forecast_start_date).isoformat()
-
-        print(forecast_start_timestamp)
-        result = get_next_1_week_forecasts_for_channel(
-            device_channel_id, forecast_start_timestamp)
+        site_id = request.args.get('site_id', type=str)
+        result = get_forecasts(site_id, db_name='daily_forecasts')

```

Comment:
don't think 'daily_forecasts' is a db_name, might need to reconsider changing the variable name.